### PR TITLE
lease: move storage.NoLease to lease package

### PR DESCRIPTION
**Pull request by heyitsanthony**
NoLease is a special LeaseID for all leases, it's not just storage only

### CODE REVIEW

File: lease/lessor.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ import (
 )
 
 var (
+	NoLease      = LeaseID(0)
```
Comment (jonboulle) [inline lease/lessor.go]: docstring exported vars, please

Comment (heyitsanthony) [inline lease/lessor.go]: ok

**Comment by xiang90** [comment]
LGTM asides from the docstring. Make golint happier.

**Comment by xiang90** [comment]
LGTM

**Comment by heyitsanthony** [comment]
will merge after CI greenlight

**Pull request merged**